core/modules/views/plugins/views_plugin_display_page.inc

```diff
@@ -264,7 +264,7 @@ class views_plugin_display_page extends views_plugin_display {
     $options['path'] = array(
       'category' => 'page',
       'title' => t('Path'),
-      'value' => views_ui_truncate($path, 24),
+      'value' => $path,
```

Comment:
Is this change from another issue?

Reply:
yes. I think this is on my 1.x branch, it keeps getting into other things...

Reply:
found it.